medium.go

```diff
@@ -214,16 +240,49 @@ func (m *Medium) ExchangeRefreshToken(rt string) (AccessToken, error) {
 
 // GetUser gets the profile identified by the current AccessToken.
 // This requires m.AccessToken to have the BasicProfile scope.
-func (m *Medium) GetUser() (*User, error) {
-	r := clientRequest{
-		method: "GET",
-		path:   "/v1/me",
+func (m *Medium) GetUser(userID string) (*User, error) {
```

Comment:
nit: could you update the header comment to indicate that GetUser will get the specified user or the current user if userID is empty?